### ISSUE TITLE
Some suggestions for improvements

### ISSUE THREAD

**Issue by paulovieira** [8bitgentleman/roam-depot-copy-code-block#4]
This is a really handy extension! Congratulations!

A few suggestions to improve:

1) The icon looks nice if you have 1 or 2 code blocks. But when you have dozens of snippets and inliners, all those icon  seem to have a bit too much dominance in the Roam look.

One simple way to mitigate would be to have an option to define the icon size ("normal", "small", etc). 

Other way would be to change the "copy icon" for something that has a more "light look" (I mean, not filled). For instance, the icon to copy the SHA in github's commit history is better, I think:

![Screenshot_2022-08-31_10-11-39](https://user-images.githubusercontent.com/2184309/187643154-33b7d661-e349-4e57-a49d-7cd7d8de7ad1.png)

In the Blueprint toolkit this seems to be called "duplicate". 

2) This is related to #2. It's funny because I actually think this button is useful for inline code blocks, but not so much for "normal" code blocks. 

My usage is more like this:
- I use inline code blocks for one liners (usually one shell command with a bunch of options which I never remember). I want to copy   and reuse it directly in the shell.
- I use "normal" code blocks for situations where I have a bunch of related stuff (for instance, a list of related shell commands, where one of them might be used or not, depending on the situation). In those I rarely want to copy everything. And it's easy to select everything anyway (which cannot be said for inline code blocks, so the extension is very useful there).

In conclusion: it would be nice if there was an extra option to enable/disable separately for inline code blocks and normal code blocks. In my case, I would have it enabled for inline and disabled for the other.

Thanks.
